packages/builder/src/components/design/settings/controls/FieldConfiguration/FieldConfiguration.svelte

```diff
@@ -152,6 +172,25 @@
 </script>
 
 <div class="field-configuration">
```

Comment:
Seemed nicer to just have two lists that conditionally hide certain fields rather than injecting a bunch of grid specific logic into the field configuration component, open to suggestions though.